### PR TITLE
Blacksmithing tongs and cooking tongs are two diffrent things. 

### PR DESCRIPTION
**Pull request by GOFLUMPYOURSELFPORCAY**
#### Summary
Content "Added blacksmithing tongs and removed cooking tongs from blacksmithing recpies."

Adds new flatjaw tongs to the game, and changes the zweihander recpie to require it. 

#### Purpose of change

Cooking tongs are really thin sheet metal that would bend/not actually be able to carry something that is being forged. They would probably bend or even break when holding hot metal.

#### Describe the solution

Added a blacksmith's flatjaw tong. 

#### Describe alternatives you've considered

We could add v-bit tongs instead? ¯\_(ツ)_/¯

### CODE REVIEW

File: data/json/recipes/weapon/cutting.json [hunk 1]
```diff
@@ -551,7 +551,7 @@
     ],
     "using": [ [ "blacksmithing_standard", 16 ], [ "steel_standard", 4 ] ],
     "qualities": [ { "id": "CHISEL", "level": 3 } ],
-    "tools": [ [ [ "hotcut", -1 ] ] ],
+    "tools": [ [ [ "hotcut", -1 ],[ "metalworking_tongs", -1 ]]],
```
[the remark this answers is not in the document]
Comment (GOFLUMPYOURSELFPORCAY) [reply data/json/recipes/weapon/cutting.json:554]: How can i do that? Sorry for asking, I'm new.

[the remark this answers is not in the document]
Comment (Night-Pryanik) [reply data/json/recipes/weapon/cutting.json:554]: Find file `\data\json\requirements\toolsets.json` and find `blacksmithing_standard` requirement there. Remove `tongs` and add your `metalworking_tongs` from there.

File: data/json/recipes/weapon/cutting.json [hunk 2]
```diff
@@ -551,7 +551,7 @@
     ],
     "using": [ [ "blacksmithing_standard", 16 ], [ "steel_standard", 4 ] ],
     "qualities": [ { "id": "CHISEL", "level": 3 } ],
-    "tools": [ [ [ "hotcut", -1 ] ] ],
+    "tools": [ [ [ "hotcut", -1 ],
```
Comment (Night-Pryanik) [inline data/json/recipes/weapon/cutting.json:554]:
```suggestion
    "tools": [ [ [ "hotcut", -1 ] ] ],
```

**Comment by Night-Pryanik** [comment]
Change `Category "Content"` in your OP to something like `Content "Added blacksmithing tongs."`

**Comment by Night-Pryanik** [comment]
You also must make these tongs spawn somewhere and probably also add a recipe for them. Otherwise there isn't a way for player to obtain them.

**Comment by GOFLUMPYOURSELFPORCAY** [comment]
> You also must make these tongs spawn somewhere and probably also add a recipe for them. Otherwise there isn't a way for player to obtain them.

Are there any blacksmith shops in game? i could add a recipe for it.

**Comment by Maleclypse** [comment]
> > You also must make these tongs spawn somewhere and probably also add a recipe for them. Otherwise there isn't a way for player to obtain them.
> 
> Are there any blacksmith shops in game? i could add a recipe for it.

There are a couple of locations that have anvils spawn.  I'd add it as a potential drop to those locations.

**Comment by GOFLUMPYOURSELFPORCAY** [comment]
> > > You also must make these tongs spawn somewhere and probably also add a recipe for them. Otherwise there isn't a way for player to obtain them.
> > 
> > 
> > Are there any blacksmith shops in game? i could add a recipe for it.
> 
> There are a couple of locations that have anvils spawn. I'd add it as a potential drop to those locations.

Added them in now, should be working! :D

**Comment by Elgrosito** [comment]
You'd need another tool to make the hole in the center, you can use a drill (cold) or something like a drift (hot).

**Comment by GOFLUMPYOURSELFPORCAY** [comment]
> You'd need another tool to make the hole in the center, you can use a drill (cold) or something like a drift (hot).

Doesn't it have a chisel already?

File: data/json/recipes/tools/tools_hand.json [hunk 3]
```diff
@@ -52,6 +52,34 @@
     ],
     "components": [ [ [ "steel_lump", 1 ], [ "steel_chunk", 4 ], [ "scrap", 12 ], [ "pipe", 3 ], [ "pipe_fittings", 1 ] ] ]
   },
+{
```
Comment (kevingranade) [inline data/json/recipes/tools/tools_hand.json:55]:
```suggestion
  {
```

File: data/json/recipes/tools/tools_hand.json [hunk 4]
```diff
@@ -52,6 +52,34 @@
...
+   "components": [
+        [
+            ["steel_lump", 1],
+            ["pipe", 2],
+            ["rebar", 2]
+        ],
+        [
+            ["nail", 1]
+        ]
+    ]
```
Comment (kevingranade) [inline data/json/recipes/tools/tools_hand.json:81]:
```suggestion
   "components": [ [ [ "steel_lump", 1 ], [ "pipe", 2 ], [ "rebar", 2 ] ], [ [ "nail", 1 ] ] ]
```

File: data/json/recipes/tools/tools_hand.json [hunk 5]
```diff
@@ -52,6 +52,25 @@
...
+    "time": "60 m",
+    "autolearn": true,
+    "qualities": [ { "id": "ANVIL", "level": 1 }, { "id": "HAMMER", "level": 2 } ],
+    "tools": [ [ [ "forge", 150 ], [ "oxy_torch", 30 ] ] ],
+    "proficiencies": [
+      { "proficiency": "prof_metalworking" },
+      { "proficiency": "prof_blacksmithing" },
+      { "proficiency": "prof_toolsmithing" }
+    ],
+   "components": [ [ [ "steel_lump", 1 ], [ "pipe", 2 ], [ "rebar", 2 ] ], [ [ "nail", 1 ] ] ]
```
Comment (kevingranade) [inline data/json/recipes/tools/tools_hand.json:72]:
```suggestion
    "components": [ [ [ "steel_lump", 1 ], [ "pipe", 2 ], [ "rebar", 2 ] ], [ [ "nail", 1 ] ] ]
```

**Pull request merged**